ft/onto/base_ontology.py

```diff
@@ -476,6 +477,16 @@ def __init__(self, pack: DataPack, begin: int, end: int):
         super().__init__(pack, begin, end)
 
 
+@dataclass
+class Body(Annotation):
```

Comment:
These does not belong to this PR, remove them

Reply:
> Don't use the word "fixes" in PR, otherwise, it will close the PR. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

I think you mentioned something like `partially fix` in the meeting. I checked the link it seems like all keywords would close the PR. Should I avoid using the keywords like `partially correct`?

Reply:
You can use any verbs outside the list:
close
closes
closed
fix
fixes
fixed
resolve
resolves
resolved